src/isPalindrome.js

```diff
@@ -19,7 +19,11 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const characters = /[^A-Za-z0-9]/g;
```

Comment:
It is not characters yet, it is just a pattern.
```suggestion
  const pattern = /[^A-Za-z0-9]/g;
```